src/themes/dark.scss

```diff
@@ -1,4 +1,21 @@
 .wxc-theme-dark {
   --wxc-meeting-background: linear-gradient(180deg, #1A1A1A 0%, #0F0F0F 100%);
   --wxc-meeting-text-color: #ffffff;
+  // Icons
+  --wxc-icon--audio-microphone--active-color: #27A17A;
+  --wxc-icon--audio-microphone--color: #fff;
+  --wxc-icon--audio-microphone-muted--color: #F55868;
+  --wxc-icon--camera--active-color: #27A17A;
+  --wxc-icon--camera-muted--color: #F55868;
+  --wxc-icon--camera--color: #fff;
+  --wxc-icon--settings--color: #fff;
+  --wxc-icon--share-screen--color: #fff;
+  --wxc-icon--cancel--color: #fff;
+
+  .wxc-old-icon {
```

Comment:
let's have them a bit more compact, there will plenty of other stuff in this file:
```css
.wxc-old-icon { display: none; }
.wxc-icon { display: block; }
```